quantum/pointing_device/pointing_device_scroll.c

```diff
@@ -0,0 +1,241 @@
+    }
+    set_scroll_mode(local_scroll_record.mode_tg);
+}
+
+/* access current toggled mode */
+uint8_t get_scroll_mode_tg(void) {
+    return local_scroll_record.mode_tg;
+}
+
+/* Change scroll mode while held */
+void scroll_key_momentary(uint8_t scroll_mode, keyrecord_t* record) {
+    if (record->event.pressed) {
+        set_scroll_mode(scroll_mode);
+    } else {
+        if (get_scroll_mode() == scroll_mode) set_scroll_mode(get_scroll_mode_tg());
+    }
+}
+
+/* handle scroll mode puttons: toggle*/
+void scroll_key_toggle(uint8_t scroll_mode, keyrecord_t* record) {
+    if (record->event.pressed) toggle_scroll_mode(scroll_mode);
+}
+
+void scroll_key_tap_toggle(uint8_t scroll_mode, keyrecord_t* record) {
+#    if TAPPING_TOGGLE != 0 && !defined(NO_ACTION_TAPPING)
+    if (record->event.pressed) {
+        set_scroll_mode(scroll_mode);
+    } else {
+        if (record->tap.count == TAPPING_TOGGLE) {
+            toggle_scroll_mode(scroll_mode);
+        } else {
+            if (get_scroll_mode() == scroll_mode) set_scroll_mode(get_scroll_mode_tg());
+        }
+    }
+#    else
+    scroll_mode_momentary(scroll_mode, record);
+#    endif
+}
+
+/* momentary scroll mode while holding  */
+void scroll_key_with_hold(uint8_t scroll_mode, uint16_t kc_hold, keyrecord_t* record) {
+    if (record->event.pressed) {
+        register_code16(kc_hold);
+        set_scroll_mode(scroll_mode);
+    } else {
+        if (get_scroll_mode() == scroll_mode) set_scroll_mode(get_scroll_mode_tg());
+        unregister_code16(kc_hold);
+    }
+}
+
+/* weak function scroll value accumulation */
+__attribute__((weak)) void pointing_device_axes_conv(scroll_record_t* scroll_record, report_mouse_t* mouse_report) {
+    // accumulate scroll if scroll key is pressed, inverting if set
+#    ifdef POINTING_DEVICE_INVERT_H
+    scroll_record->h -= mouse_report->x;
+#    else
+    scroll_record->h += mouse_report->x;
+#    endif
+
+#    ifdef POINTING_DEVICE_INVERT_V
+    scroll_record->v -= mouse_report->y;
+#    else
+    scroll_record->v += mouse_report->y;
+#    endif
+    // stop cursor movement
+    mouse_report->x = mouse_report->y = 0;
+}
+
+/* process scroll modes */
+static void process_scroll_mode(scroll_record_t* scroll_record, report_mouse_t* mouse_report) {
+    if (!process_scroll_mode_kb(scroll_record, mouse_report)) return;
+
+    switch (scroll_record->mode) {
+        case SM_DRAG:
+            // Ensure larger than divisor to avoid clearing unless actually written
+            if (abs(scroll_record->h) > SCROLL_DRAG_DIVISOR) {
+                mouse_report->h = pointing_device_hv_clamp(scroll_record->h / SCROLL_DRAG_DIVISOR);
+                scroll_record->h %= SCROLL_DRAG_DIVISOR;
+            }
+            if (abs(scroll_record->v) > SCROLL_DRAG_DIVISOR) {
+                mouse_report->v = pointing_device_hv_clamp(scroll_record->v / SCROLL_DRAG_DIVISOR);
+                scroll_record->v %= SCROLL_DRAG_DIVISOR;
+            }
```

Comment:
What's wrong with `>=` here?

Reply:
Nothing I likely forgot or somehow thought not checking equal would be slightly faster?
Either way checking equal makes it more consistent so I will fix it.